aries-site/src/examples/components/layouts/LayerExample.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext, useState } from 'react';
+import { Button, Box, Heading, Layer, ResponsiveContext, Text } from 'grommet';
+import { Close } from 'grommet-icons';
+
+export const LayerExample = () => {
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  return (
+    <>
+      <Box align="start">
+        <Button
+          color="blue"
+          label="Show me the Layer"
+          onClick={onOpen}
+          primary
+        />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={size !== 'small' ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={size !== 'small' ? 'medium' : undefined}
+            pad="medium"
+          >
+            <Box flex={false} direction="row" justify="between">
+              <Heading level={2} margin="none">
+                This is a Layer.
+              </Heading>
+              <Button icon={<Close />} onClick={onClose} />
+            </Box>
+            <Box flex="grow" overflow="auto" pad={{ vertical: 'medium' }}>
```

Comment:
Not a code issue, but from a UX perspective the content and the close button are quite far apart. Do we need the flex="grow" here? Does the button pushed to the bottom add value?

Reply:
Agreed -- it is a bit far from the content. Was pulling from a grommet storybook example which had a bit more of a form here. I think bringing the button up would be fine.